iOS/Flows/Conversation/Cells/EmotionCircleCollectionViewDataSource.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  EmotionCircleCollectionViewDataSource.swift
+//  Jibber
+//
+//  Created by Martin Young on 4/6/22.
+//  Copyright © 2022 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+
+typealias EmotionCircleSection = EmotionCircleCollectionViewDataSource.SectionType
+typealias EmotionCircleItem = EmotionCircleCollectionViewDataSource.ItemType
+
+class EmotionCircleCollectionViewDataSource: CollectionViewDataSource<EmotionCircleSection,
+                                                EmotionCircleItem> {
```

Comment:
Pretty basic datasource. Nothing unusual here.

For now the emotion item doesn't have a count. Adding that soon.